crates/pop-parachains/src/templates.rs

```diff
@@ -90,15 +66,15 @@ pub struct Config {
 	PartialEq,
 	VariantArray,
 )]
-pub enum Template {
+pub enum ParachainTemplate {
 	/// Minimalist parachain template.
 	#[default]
 	#[strum(
 		serialize = "standard",
 		message = "Standard",
 		detailed_message = "A standard parachain",
 		props(
-			Provider = "Pop",
+			TemplateType = "Pop",
```

Comment:
Naming was better before imo. I think just Type might be okay, but a better solution might be to add a const on the trait, with a default of "Type", so that the implementor can define the value to be used.